pkg/operator/controller/ingress/load_balancer_service_test.go

```diff
@@ -344,6 +337,9 @@ func Test_desiredLoadBalancerService(t *testing.T) {
 				t.Errorf("expected desiredLoadBalancerService to return nil service for endpoint publishing strategy type %v, got %#v", tc.strategyType, svc)
 			}
 
+			if tc.strategyType != operatorv1.LoadBalancerServiceStrategyType {
```

Comment:
It's a minor thing, but if I were doing this over again, I'd check `tc.expect` and `haveSvc` instead of the strategy:
```suggestion
			if !tc.expect || !haveSvc {
```

Reply:
done.